README.md

```diff
@@ -1,10 +1,23 @@
-Apache Cassandra website builder
-================================
+Apache Cassandra website
+========================
+
+Development Cycle
+-----------------
+
+Making changes to the website is done with the following steps.
+
+1. Test changes locally
+2. Commit changes in `src/` to a fork and branch, and create a pull request
+3. Get the pull request reviewed and merged to `master`
+4. Test on https://cassandra.staged.apache.org/ (wait til [ci-cassandra.apache.org](https://ci-cassandra.apache.org/job/cassandra-website/) has deployed it)
+5. Merge `asf-staging` to `asf-site`
+6. Test on https://cassandra.apache.org/
 
-Building the Apache Cassandra website takes a number of steps. To make things easier we have provided a Docker container which can build the full website in two simple commands and have it ready to commit via git. If you are interested in the process head over to the [README](./src/README) in _src_ directory.
 
-Prerequisite
-------------
+To test changes before committing, it is required to build the website locally. Building the Apache Cassandra website takes a number of steps. To make things easier we have provided a Docker container which can build the full website in two simple commands and have it ready to commit via git. If you are interested in the process head over to the [README](./src/README) in _src_ directory.
```

Comment:
Might need to reword the first sentence. Specifically add a few words to the part after the comma. Feel free to use any of the following variations:

> To test changes before committing, it is a requirement that you build the website locally.

OR

> To test changes before committing, it is required that you build the website locally.

OR (completely reword it)

> To test changes before committing, you will need to build the website locally.